api/helpers/utils.js

```diff
@@ -347,6 +347,19 @@ exports.filterData = function (collection, data, roles) {
       delete item.reviewExtensions;
     });
     return data;
+  } else if (collection === 'Organization') {
+    _.each(data, function (item) {
+      console.log(item);
```

Comment:
If you need the console out for some reason, set it to use the default logger (defaultLog.debug or whatever). Looks like the log out can probably just be cut though